src/rewards/rewards.controller.ts

```diff
@@ -33,4 +41,28 @@ export class RewardController {
   ): Promise<RewardMerkleClaimModel> {
     return this.rewardsService.getUserRewards(Chain.getChain(chain), address);
   }
+
+  @Get('/schedules/vault/:address')
```

Comment:
all schedules wil be for vaults, we can probably just move this to `/schedules/:address`

Reply:
make sense, will correct it